### ISSUE TITLE
Future Plans

### ISSUE THREAD

**Issue by ludos1978** [VCL3D/VolumetricCapture#99]
Hi, i wonder what the plans for VCL3D are? 

- According to the issues it's development is abandoned. 
- It's source code remains closed. 
- The pipeline only works to record, but there is no playback possible.
- Bug's start to appear because the underlying os is changing.

It would be very bad if such a nice project would be lost. Also because it's a Horizon funded Project, which should permit and promote further research, is most likely going to be lost.

Could you reconsider to release it's source?

**Comment by ankarako** [comment]
Hi @ludos1978 ,

Thanks for this. Actually right now we are implementing an updated version (as part of another project) and probably it will be released (in the same manner, i.e. as binaries, and a playback application can be added too).
Unfortunately, releasing the source-code is restricted by our organization.

Best

**Comment by brooklynlittell** [comment]
Hi @ankarako ,

Do you have any updates on the status of the updated version? The current version crashes on application boot and there's not much to investigate since it's closed source.

Thanks :smile:

**Comment by ankarako** [comment]
Hi @brooklynlittell,

It will be sometime until we release the next version. Please try to run the executable via the command line and check if there are errors shown.

**Comment by seltzdesign** [comment]
Yes, what a shame. The software looks amazing and exactly what we need, but it seems like it is not really working any longer. What would it take to get an update or is it the companies property?

**Comment by ankarako** [comment]
[new version](https://vcl3d.github.io/VolumetricCapture/docs/v2_dual_volcap/)
Not that we will (again) be able to help a lot or update, but...
